patches/minecraft/net/minecraft/server/management/PlayerInteractionManager.java.patch

```diff
@@ -271,17 +264,20 @@
                  }
              }
          }
-@@ -453,4 +498,13 @@
+@@ -453,4 +498,16 @@
      {
          this.field_73092_a = p_73080_1_;
      }
 +
++    @Deprecated // use the attribute directly
```

Comment:
Why not keep and use `getBlockReachDistance` the way it's defined now? It would reduce the number of patches.

Reply:
Deprecating this and moving all access to the attribute allows for our old patches to die off in 1.13 and everything to access stuff in one place.